jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -332,7 +332,38 @@ def _uncertainty(result):
         for function in FUNCTIONS:
             # TODO: update specutils to allow ALL analysis to take regions and continuum so we
             # don't need these if statements
-            if function == "Equivalent Width":
+            if function == "Line Flux":
+                meta = self.dataset.selected_dc_item.meta
+                telescope = meta.get('telescope', meta.get('TELESCOP', ''))
```

Comment:
Is this future proofing? AFAIK, only Imviz knows how to load ASDF metadata. For FITS, it is always TELESCOP because FITS keyword cannot be longer than 8 characters.

Reply:
Figured it was a low-cost way to future proof. But since FITS is the pathway for now, maybe I should swap the order of the two?